src/game.js

```diff
@@ -1,38 +1,56 @@
-const dbGame = [];
-const idHelper = require('./IdHelper.js');
+const idHelper = require("./IdHelper.js");
+const GameData = require("./data-base/GameData");
 
 class Game {
-	constructor({cols = 10, rows = 10} = {}){
+  constructor({ cols = 10, rows = 10 } = {}) {
     this.cols = cols;
-		this.rows = rows;
-  };
-  
-	static create({cols = 10, rows = 10} = {}) {
-		const game = new Game({cols,rows});
-		game.id = dbGame.length + 1;
-		game.playerId = idHelper();
-		const token = idHelper();
-		game.token = token
-		dbGame.push(game);
-		game.session = `http://localhost:3000/game?token=${token}`;
+    this.rows = rows;
+  }
 
-		return Promise.resolve({
-			id : game.id, 
-			session : game.session,
-			playerId : game.playerId
-		})
-	};
+  static create({ cols = 10, rows = 10 } = {}) {
+    const game = new Game({ cols, rows });
+    game.playerId = idHelper();
+    const token = idHelper();
+    game.token = token;
+    game.session = `http://localhost:3000/game?token=${token}`;
+    console.log(game.cols);
+    console.log(game.rows);

```

Comment:
remove console logs (they are allowed only to print error or api messages, not for debugging purposes)

Reply:
it was removed